explorer/client/src/__tests__/e2e.test.ts

```diff
@@ -53,6 +53,19 @@ const searchText = async (page: any, text: string) => {
     await page.click('#searchBtn');
 };
 
+//Standardized CSS Selectors
+
+const coinGroup = (num: number) => {
```

Comment:
cool! i was just expecting a simple string constant for the selector's base, but this is even better.